src/main/java/mops/persistence/file/FileInfoBuilder.java

```diff
@@ -225,17 +231,21 @@ public FileInfoBuilder tags(@NonNull Iterable<String> tags) {
     @SuppressWarnings("PMD.CyclomaticComplexity") //if-else chain necessary for fine grained exception messages
     public FileInfo build() {
         if (name == null) {
+            log.error("Directory is not completely setup name was not set.");
             throw new IllegalStateException("FileInfo incomplete: name must be set!");
         } else if (directoryId == -1L) {
+            log.error("Directory is not completely setup directory id not set.");
             throw new IllegalStateException("FileInfo incomplete: directoryId must be set!");
         } else if (type == null) {
+            log.error("Directory is not completely setup type was not set.");
             throw new IllegalStateException("FileInfo incomplete: type must be set!");
         } else if (size == -1L) {
+            log.error("Directory is not completely setup size was not set.");
             throw new IllegalStateException("FileInfo incomplete: size must be set!");
         } else if (owner == null) {
+            log.error("Directory is not completely setup owner was not set.");
```

Comment:
grammar?

Reply:
Should be something along the lines of:
`"Directory is not setup completely: owner was not set."`
or:
`"Directory is not complete: owner was not set."`